runtime/rasdump/trigger.c

```diff
@@ -1002,7 +1002,35 @@ triggerDumpAgents(struct J9JavaVM *vm, struct J9VMThread *self, UDATA eventFlags
 								OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
 								char dateStamp[64];
 								omrstr_ftime_ex(dateStamp, sizeof(dateStamp), "%Y/%m/%d %H:%M:%S", now, OMRSTR_FTIME_FLAG_LOCAL);
-								j9nls_printf(PORTLIB, J9NLS_INFO | J9NLS_STDERR | J9NLS_VITAL, J9NLS_DMP_PROCESSING_EVENT_TIME, mapDumpEvent(eventFlags), detailLength, detailData, dateStamp);
+
+								/*If there is more details about the event, print it.
+								 * During abort event detailData is empty string - skip the abort event */
+								if (0 != strcmp(detailData,"") && NULL != eventData->exceptionRef && NULL != *eventData->exceptionRef) {
```

Comment:
Pls add a space after the comma, and bracket the two conditions
i.e.
`strcmp(detailData, "")`
`(NULL != eventData->exceptionRef)`